esignet-service/internal/keymanager/kmtest/kmtest.go

```
@@ -0,0 +1,292 @@
+	mu      sync.Mutex
+	aliases map[string][]db.KeyAlias // key: appID+"|"+refID, index 0 = most recently inserted
+}
+
+// NewStateQuerier constructs an empty StateQuerier.
+func NewStateQuerier() *StateQuerier {
+	return &StateQuerier{aliases: map[string][]db.KeyAlias{}}
+}
+
+func aliasMapKey(appID, refID string) string { return appID + "|" + refID }
+
+func (q *StateQuerier) GetKeyAliasesByAppRef(_ context.Context, appID, refID string) ([]db.KeyAlias, error) {
```

golangci-lint (2.12.2)

[warning] exported: exported method StateQuerier.GetKeyAliasesByAppRef should have comment or be unexported

(revive)